composer.json

```diff
@@ -0,0 +1,62 @@
+{
+    "name": "openeuropa/oe_editorial",
+    "description": "OpenEuropa Drupal module template.",
+    "type": "drupal-module",
+    "license": "EUPL-1.2",
+    "minimum-stability": "dev",
+    "prefer-stable": true,
+    "require": {
+        "openeuropa/drupal-core-require": "^8.6",
```

Comment:
Let's remove this please. We can directly use drupal/core

Reply:
Removing this dependency is not in the scope of this ticket.
We should create a follow-up ticket and include the changes in the template drupal module.

Reply:
OK, fair enough. Can you please create the followup?